coffee-grpc/coffee-grpc-client-extension/src/main/java/hu/icellmobilsoft/coffee/grpc/client/interceptor/ClientResponseInterceptor.java

```diff
@@ -52,13 +59,47 @@ public ClientResponseInterceptor() {
     @Override
     public <ReqT, RespT> ClientCall<ReqT, RespT> interceptCall(MethodDescriptor<ReqT, RespT> method, CallOptions callOptions, Channel next) {
 
-        ClientCall<ReqT, RespT> call = new SimpleForwardingClientCall<ReqT, RespT>(next.newCall(method, callOptions)) {
+        final String configKey = callOptions.getOption(GrpcClientProducerFactory.CONFIG_VALUE_KEY);
+
+        return new SimpleForwardingClientCall<>(next.newCall(method, callOptions)) {
+
             @Override
             public void start(Listener<RespT> responseListener, Metadata headers) {
-                Listener<RespT> listener = new SimpleForwardingClientCallListener<RespT>(responseListener) {
+
```

Comment:
ide is jó lenne null check a bejövő paraméterekre.

Reply:
A responseListener-t szerintem felesleges, csak továbbítjuk. 
A headers-t hasonlóan.